e2e/explorer/workflow/active.spec.ts

```diff
@@ -1 +1,138 @@
-// add tests for /namespace/explorer/workflow/active tab here.
+import { Page, expect, test } from "@playwright/test";
+import {
+  createNamespace,
+  deleteNamespace,
+} from "../../utils/namespace";
+
+import {
+  createWorkflow,
+} from "../../utils/node";
+import { faker } from "@faker-js/faker";
+
+let namespace = "";
+let workflow = "";
+const defaultDescription = 'A simple \'no-op\' state that returns \'Hello world!\''
+
+test.beforeEach(async ({page}) => {
+  namespace = await createNamespace();
+  workflow = await createWorkflow(namespace, faker.git.shortSha() + '.yaml');
+  test.setTimeout(120000)
+  await page.goto("/");
```

Comment:
It is good to have one test that navigates to the revision from the namespace. But I think it's not ideal to repeat all these steps in the beforeTest() hook. When we add more tests, the time that this takes will add up. I think it's better to keep the beforeTest() short, it should focus on setup and not include too many assertions.

I think it would be better to have one test for only "it is possible to navigate to the active revision", then every test for specific functions can start by visiting `${namespace}/explorer/workflow/active/${workflow}`.